src/software/parameter/config/full_system_main_command_line_args.yaml

```diff
@@ -0,0 +1,17 @@
+FullSystemMainCommandLineArgs:
+    backend:
+        type: "std::string"
+        default: ""
+        description: >-
+            The name of the backend you would like to use. See /software/backend for options
+            ex. class WifiBackend this arugment would be WfiBackend
+    interface:
+        type: "std::string"
+        default: ""
+        description: >-
+            The interface to send and receive packets over (can be found through ifconfig)
+    headless:
```

Comment:
Wondering if "headless" is the best term to describe running without FullSystemGui.

Reply:
it's a pretty common term, e.g. webdrivers

Reply:
yeah seems fairly standard, we can probably keep it as is

https://en.wikipedia.org/wiki/Headless_software